main/src/library/pkger/pkger.flix

```diff
@@ -57,9 +55,8 @@ pub def initPackage(name: String): /* Package */ Result[Path, ErrorFull] & Impur
     let projectDir = map(getProjectDir, workingDir); // Result[Path, IOError]
     let projectDirExists = flatMap(Path.exists, projectDir); // Result[Bool, IOError]
 
-    let test = (b: Path) -> { projectDir as & Impure }; // Path -> Result[Path, IOError]
+    let test = (_) -> { projectDir as & Impure }; // a -> Result[Path, E]
```

Comment:
I've spend quite a bit of time on these not very many lines of code, and I'm still utterly baffled as to what the issue is. Since I can't annotate with the `IOError` type I've written the annotations as comments. Right now it's trying to unify `Bool` and `Path`, but after trying several combinations I'm just utterly confused. `test` should, as far as I understand, take a `Bool` based on the surrounding program. But if I type it as a `Bool` it complains that `Path` is needed.

Maybe you can see where I'm taking a wrong step. I've been staring myself blind on this one.